go/uidmap/uidmap_test.go

```diff
@@ -177,14 +177,15 @@ func TestRanOutOfTime(t *testing.T) {
 
 	// now success for user t_kb, who has a non-hardcoded username and a fullname on the
 	// server
+	t.Logf("tKB: %s", tKB)
 	uids = []keybase1.UID{tKB}
 	hit = false
 	results, err = uidMap.MapUIDsToUsernamePackages(context.TODO(), tc.G, uids, 0, 0, true)
 	require.NoError(t, err)
 	require.True(t, hit)
 	require.Equal(t, results[0].NormalizedUsername, libkb.NewNormalizedUsername("t_kb"))
 	require.Equal(t, results[0].FullName.FullName, keybase1.FullName("Joe Keybaser"))
-	require.Equal(t, results[0].FullName.EldestSeqno, keybase1.Seqno(1))
```

Comment:
`uidmap` has also not been run in Jenkins, and this test is just broken I think. These users have an eldest of 0 not 1.